Python/Tests/TestData/TestExecutor/test_cancel.py

```diff
@@ -0,0 +1,18 @@
+import unittest
+import time
+
+class CancelTests(unittest.TestCase):
+    def test_sleep_1(self):
+        time.sleep(0.1)
+
+    def test_sleep_2(self):
+        time.sleep(5)
+
+    def test_sleep_3(self):
+        time.sleep(5)
```

Comment:
why is sleep_3 the same as sleep_2?  is that seconds?

Reply:
yes in seconds. as discussed privately, there are 2 long running tests to give enough time for us to cancel.